types/mermaid/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "exports": {
+        ".": {
+            "types": {
+                "import": "./index.d.mts",
+                "default": "./index.d.ts"
+            }
+        }
+    }
+}
```

Comment:
Using a special `index.d.mts` file and package exports technically matches the mermaid files, but mayb it can be omitted, since the `node16` module resolution can use default imports to import modules defined with `require =`.